src/App.vue

```diff
@@ -2,37 +2,107 @@
   <v-app>
     <v-toolbar app>
       <v-toolbar-title class="headline text-uppercase">
-        <span>Vuetify</span>
-        <span class="font-weight-light">MATERIAL DESIGN</span>
+        <span>Posts</span>
       </v-toolbar-title>
       <v-spacer />
-      <v-btn
-        flat
-        href="https://github.com/vuetifyjs/vuetify/releases/latest"
-        target="_blank"
-      >
-        <span class="mr-2">Latest Release</span>
-      </v-btn>
     </v-toolbar>
-
     <v-content>
-      <HelloWorld />
+      <v-card>
+        <v-container
+          fluid
+          grid-list-lg
+        >
+          <v-layout>
+            <v-flex>
+              <v-text-field
+                v-model="searchText"
+                label="Post Id"
+                single-line
+                outline
+              />           
+            </v-flex>
+            <v-flex> 
+              <v-btn
+                large 
+                color="primary"
+                @click="loadPost"
+              >
+                Load
+              </v-btn>
+            </v-flex>
+          </v-layout>
+          <v-layout 
+            v-for="(post, index) in posts"
+            :key="index"
+          >
+            <v-flex xs12>
+              <v-card>
+                <v-card-title primary-title>
+                  <div>
+                    <div>
+                      Post Id: {{ post.id }}
+                    </div>
+                    <div class="headline">
+                      {{ post.title }}
+                    </div>
+                    <span>
+                      {{ post.body }}
+                    </span>
+                  </div>
+                </v-card-title>
+              </v-card>
+            </v-flex>
+          </v-layout>
+        </v-container>
+      </v-card>
     </v-content>
   </v-app>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld'
+import axios from "axios";
```

Comment:
Pull Axios into its own js file under an "assets" or "library" folder. For instance "src/assets/api.js"

Export the various calls from there so they can then be used in the Vue component by just calling the relevant function.

See here: https://medium.com/canariasjs/vue-api-calls-in-a-smart-way-8d521812c322